### PR TITLE
Adding dynaconf module

### PR DESCRIPTION
**Pull request by GeorgeJuniorGG**
Ping @hyviquel

### CODE REVIEW

**Review by tldahlgren — COMMENTED** [review]
Confirmed the homepage and version sha256.  Adding @adamjstewart since he's the python package maintainer and more thorough regarding dependencies, etc.

File: var/spack/repos/builtin/packages/py-dynaconf/package.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+class PyDynaconf(PythonPackage):
+    """Dynaconf is a dynamic configuration management package for Python projects"""
+
+    homepage = "https://github.com/dynaconf/dynaconf"
+    pypi = "dynaconf/dynaconf-3.2.2.tar.gz"
+
+    version("3.2.2", sha256="2f98ec85a2b8edb767b3ed0f82c6d605d30af116ce4622932a719ba70ff152fc")
+
+    depends_on("python@3.8:", type=("build", "run"))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-dynaconf/package.py:17]: Missing build dep:
```python
    setup_requires=["setuptools>=38.6.0"],                                               
```

**Review by adamjstewart — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/py-dynaconf/package.py (hunk 1 above)
Comment (GeorgeJuniorGG) [reply var/spack/repos/builtin/packages/py-dynaconf/package.py:17]: Just added it, thanks!

**Review by tldahlgren — APPROVED** [review]
LGTM

**Comment by tldahlgren** [comment]
I'm able to successfully build the package.

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**